apps/drupal/templates/block/block--system-branding-block.html.twig

```diff
@@ -15,16 +15,15 @@
  * - site_name: Name for site as defined in Site information settings.
  * - site_slogan: Slogan for site as defined in Site information settings.
  *
- *
  */
 #}
 
 {% block content %}
 
   {# Use logo to explicitly override values from preprocess. #}
   {% set logo = {
-    logo_svg_inline: logo_inline_svg ? logo_inline_svg : '',
-    logo_img_src: logo_img_src ? logo_img_src : ''
+    logo_svg_inline: logo_inline_svg ?: logo_inline_svg,
+    logo_img_src: logo_img_src ?: logo_img_src
```

Comment:
Neither does this.

Reply:
Ah! Misread your last comment. Updating.